src/main/java/com/_up/megastore/controllers/interfaces/IUserController.java

```diff
@@ -16,8 +16,8 @@ void activateUser(@PathVariable UUID userId,
       @RequestBody @Valid ActivateUserRequest activateUserRequest);
 
   @PostMapping("/recover-password/send-email") @ResponseStatus(HttpStatus.NO_CONTENT)
-  void sendEmailToRecoverPassword(@RequestParam String email);
+  void sendEmailToRecoverPassword(@RequestBody String email);
```

Comment:
Poner un @RequestBody como String es bastante complejo, puesto que en la desserialización se agregan algunos caracteres extra. Personalmente crearía un objeto nuevo
```
public record SendRecoverPasswordEmailRequest (
     // Validaciones como @Email 
     String email
)
```
Y lo usaría como body en el método

 ```suggestion
  void sendEmailToRecoverPassword(@RequestBody SendRecoverPasswordEmailRequest sendRecoverEmailPasswordRequest);
```